core/api/usermanager/v1/usermanager_proto/usermanager.proto

```diff
@@ -36,7 +36,7 @@ message GetKeySetRequest {
 // It is the responsibility of authorized callers to verify that domain_id/app_id/user_id is correct.
 message CreateUserRequest {
   // user is the user to create.
-  keytransparency.v1.proto.User user = 2;
+  type.User user = 2;
```

Comment:
Then this would be `types.User user = 2;` which I think reads better in any case. wdyt?

Reply:
see above

Reply:
OK